README.md

```diff
@@ -1,3 +1,21 @@
+# Thai Programmers USA
+[thai-programmers-usa](https://thai-programmers-usa.github.io) is an open source application for Thai programmers in USA learn how to contribute to open source project. It is run by a nonprofit to help Thai people in USA transition into tech. You can find us on [Facebook](https://www.facebook.com/groups/thaiprogrammersinusa).
+
+### File an Issue
+
+![Bugs](https://img.shields.io/github/issues/thai-programmers-usa/thai-programmers-usa.github.io/bug?label=Bugs) ![Feature Requests](https://img.shields.io/github/issues/thai-programmers-usa/thai-programmers-usa.github.io/feature-request?color=blue&label=Feature%20Requests)
+
+## Roadmap
+
+Here's a snippet of what's currently ✅ available or ❌ unavailable and what we have on our radar for the ⏩future:
+
+Current Features| Future Roadmap |
+|:--|:--|
+❌ Payment Transaction <br>  | ⏩ Sign up/ login <br> ⏩ Profile Edit <br> ⏩ Online name cards <br> 
```

Comment:
Why Payment Transaction is listed here?

Reply:
Probably he meant show that no payment required?

Reply:
I think it means `❌ unavailable and what we have on our radar for the`

Reply:
But I don't think we have any plan around that topic yet

Reply:
You can require the change to block the merge till we got answer

Reply:
Just wanted to put something in there, probably too much misleading.

Reply:
@BayLadyCoder Those two are different sentences.
- ❌ unavailable
- , and what we have on our radar for the ⏩future:

Reply:
I see. Let's remove Payment Transaction for now. We can replace it with other topics that we might add later after this online name cards project is done such as a job board or community board.

Reply:
Or just skip unavailable topic entirely, is fine too.